### DISCUSSION TITLE
Folders to contain child docs

### DISCUSSION THREAD

**Discussion by binaryfire** [outline/outline#5093] [Ideas]
Hey Tom

Currently the only way to create "folders" in a collection is to nest documents under an empty "parent" doc. However this can be confusing for users since the parent doc can still be opened and is empty.

What do you think about adding folders?

**Comment by tommoor** [comment]
The parent doc should automatically show a list of child docs in this case, is that not happening?

**Comment by binaryfire** [comment]
It does but they're at the bottom. So the parent still seems like an empty (incomplete) page. What about moving that list to the top if the page is empty? It'd be more like a table of contents then.

**Reply by tommoor** [reply]
Yes, this used to happen (and still does) when the separate editing mode is enabled. Agreed that we need to figure out how to make this work for the always-editing mode too
